coordinator/src/dlc_protocol.rs

```diff
@@ -392,6 +393,12 @@ impl DlcProtocolExecutor {
                 .map_err(|_| RollbackTransaction)?,
         );
 
+        // TODO(holzeis): its a bit ugly to calculate the order matching fee like that. The user
+        // might get a discount at some point. So ideally the order matching fee is decided on the
+        // trade params at a single point and then only stored here.
```

Comment:
You could already pull this up and provide it as a param because we have this information available in `open_position` and `open_dlc_channel`. Only in `start_closing_position` I couldn't find it but maybe I just missed it.

Reply:
I fear it's not that simple. We'd have to add the order matching fee to the `Matches` and the `TradeParams` table and then pass it through.

This seems to be quite some changes. I'll do this in a follow up PR.

Reply:
Also that order matching fee should then be taken when calculating the payout curve.